docs/FormatsAndConventions.md

```diff
@@ -17,11 +17,16 @@ are not desirable.
 
 Every commit must have a well written commit message _**subject line**_.
  
-1. Limit the subject line to 50 characters
-1. Capitalize the subject line
+1. Try to limit the subject line to 50 characters (hard limit: 72 chars)
+
+   > Rationale: Some tools show only a limited number of characters from the commit message.
+
+1. Capitalize the subject line e.g. `Move index.html file to root`
 1. Do not end the subject line with a period
 1. Use the imperative mood in the subject line
    e.g. `Add README.md` rather than `Added README.md` or `Adding README.md`
+1. Use `scope: change` format when applicable. 
+   e.g. `Person class: remove static imports`, `Unit tests: remove blank lines`
```

Comment:
Full stop before `e.g.` or not? We'll have to be consistent for lines [24](https://github.com/oss-generic/process/pull/61/files#diff-0ec80c39f8532a34d5cc80d4123d4675R24), [26](https://github.com/oss-generic/process/pull/61/files#diff-0ec80c39f8532a34d5cc80d4123d4675R26) and [28](https://github.com/oss-generic/process/pull/61/files#diff-0ec80c39f8532a34d5cc80d4123d4675R28).